package.json

```diff
@@ -8,16 +8,27 @@
   },
   "dependencies": {
     "nan": "^2.0.4",
-    "inherit": "~2.2.2"
+    "inherit": "~2.2.2",
+    "node-pre-gyp": "0.6.x"
   },
   "devDependencies": {
     "mocha": "^2.0.1",
     "must": "~0.12.0",
-    "node-gyp": "^2.0.2"
+    "node-gyp": "^3.0.3",
+    "aws-sdk": "2.2.x"
   },
+  "bundledDependencies": ["node-pre-gyp"],
   "scripts": {
     "test": "./dev/test",
-    "build": "node-gyp rebuild"
+    "build": "node-gyp rebuild",
+    "install": "node-pre-gyp install --fallback-to-build",
```

Comment:
When downloading a binary, node-pre-gyp currently (as by default) looks for Node ABI, os and architecture.

There's a problem with pre-building 32bit linux binaries, as travis-ci only provides x64 VMs.